tests/integration/golang/aim/metric/search_test.go

```diff
@@ -5929,6 +5921,16 @@ func (s *SearchMetricsTestSuite) Test_Ok() {
 					timestampsKey := prefix + ".timestamps.blob"
 					valuesKey := prefix + ".values.blob"
 
+					contextPrefix := prefix + ".context"
+					contx, err := helpers.ExtractContextBytes(contextPrefix, decodedData)
+					s.Require().Nil(err)
```

Comment:
I think it should be a part of our standard  decoder unfortunately and work for any object and not only for `context`.

Reply:
done